ParquetCSVImporter/Import Program.cs

```diff
@@ -0,0 +1,94 @@
+using System.Text;
+using System.IO;
+using System.Collections.Generic;
+using System.Globalization;
+using CsvHelper;
+using CsvHelper.TypeConversion;
+using ParquetClassLibrary.Sandbox.ID;
+using ParquetClassLibrary.Sandbox.Parquets;
+using ParquetCSVImporter.ClassMaps;
+
+namespace ParquetCSVImporter
+{
+    /// <summary>
+    /// A program that reads in Parquet definitions from CSV files, and outputs them as JSON.
+    /// TODO Add ability to output them as JSON!
+    /// </summary>
+    class MainClass
+    {
+        /// <summary>The location of the Designer files.</summary>
+        public static readonly string SearchPath =
+#if DEBUG
+            Directory.GetParent(System.IO.Directory.GetCurrentDirectory()).Parent.Parent.FullName;
+#else
+            Environment.CurrentDirectory;
+#endif
+
+        /// <summary>All parquets defined in the CSV files.</summary>
+        public static readonly HashSet<ParquetParent> Parquets = new HashSet<ParquetParent>();
+
+        /// <summary>Instructions for handling integer type conversion when reading in parquet identifiers.</summary>
+        public static readonly TypeConverterOptions IdentifierOptions = new TypeConverterOptions
+        {
+            NumberStyle = NumberStyles.AllowLeadingSign &
+                          NumberStyles.Integer
+        };
+
+        /// <summary>
+        /// The entry point of the ParquetCVSImporter program, where the program control starts and ends.
+        /// </summary>
+        public static void Main()
+        {
+            var recordsFromCSV = new List<ParquetParent>();
+            recordsFromCSV.AddRange(GetRecordsForType<Floor>());
+            recordsFromCSV.AddRange(GetRecordsForType<Block>());
+            recordsFromCSV.AddRange(GetRecordsForType<Furnishing>());
+            recordsFromCSV.AddRange(GetRecordsForType<Collectable>());
+
+            Parquets.UnionWith(recordsFromCSV);
```

Comment:
Does this intentionally not clear out the Parquets that were there before? It's possible for someone to run this twice by calling MainClass.Main(), which would result in getting a list of Parquets twice. I could see someone needing to reload all Parquets after making some sort of change to the files.

Reply:
Good observation.  So, my intent was that this never be called except when run from the command line as a game editing tool.  But I think the precaution you suggest is sensible, so I added it in a733786.